src/lib/nodeCache.ts

```diff
@@ -0,0 +1,58 @@
+import crypto from "crypto"
+import NodeCache from "node-cache"
+import util from "util"
+
+const debugLog = util.debuglog("cache")
+
+const DEFAULT_OPTIONS = {
+  checkperiod: 120,
+  maxKeys: 10000,
+  stdTTL: 10 * 60,
+  useClones: false,
+}
+
+export class MemoryCache {
```

Comment:
If this class is called `MemoryCache` can we rename the file to `MemoryCache`? Or vice-versa.